pkg/controller/pod_driver.go

```diff
@@ -494,7 +497,15 @@ func (p *PodDriver) podReadyHandler(ctx context.Context, pod *corev1.Pod) error
 	}
 
 	e := util.DoWithTimeout(ctx, defaultCheckoutTimeout, func() error {
-		_, e := os.Stat(mntPath)
+		finfo, e := os.Stat(mntPath)
+		if e != nil {
+			return e
+		}
+		if st, ok := finfo.Sys().(*syscall.Stat_t); ok {
+			if st.Ino == 1 {
+				util.MountPointDevMinorTable.Store(mntPath, util.DevMinor(st.Dev))
```

Comment:
I worry this table could get lost between csi-node restarts, is this a concern?

Reply:
when csi-node restarts, will re-reconcile all mount pods and update this table

and will also reconcile each mountpod every 10 minutes